src/primitives/transaction.cpp

```diff
@@ -58,14 +58,11 @@ CPourTx::CPourTx(ZerocashParams& params,
     boost::array<std::vector<unsigned char>, NUM_POUR_INPUTS> serials_bv;
     boost::array<std::vector<unsigned char>, NUM_POUR_OUTPUTS> commitments_bv;
     boost::array<std::vector<unsigned char>, NUM_POUR_INPUTS> macs_bv;
-    boost::array<std::string, NUM_POUR_OUTPUTS> ciphertexts_bv;
 
-    proof = pourtx.unpack(serials_bv, commitments_bv, macs_bv, ciphertexts_bv);
+    proof = pourtx.unpack(serials_bv, commitments_bv, macs_bv, ciphertexts, ephemeralKey);
```

Comment:
I have intentionally left our hSig changes out of this PR, since they involve several other unrelated components and changes.